esx_service/vsan_policy.py

```diff
@@ -204,12 +203,12 @@ def delete(name):
 
     vmdk = policy_in_use(path, name)
     if vmdk:
-        return 'Error: Cannot remove. Policy is in use by {0}'.format(vmdk)
+        return 'Cannot remove. Policy is in use by {0}'.format(vmdk)
```

Comment:
Because this error message is prepended with 'ERROR' string when through vmdkops_admin.
This is common for every failed vmdkops admin command. So we don't need additional string.